### DISCUSSION TITLE
What version supports redis cluster

### DISCUSSION THREAD

**Discussion by mzzgaopeng** [goharbor/harbor#15871] [Ideas]
Is there any plan to say that harbor components support redis cluster mode instead of sentinel mode?

Such as three masters and three slaves or five masters and five slaves

**Comment by wy65701436** [comment]
So far, there is no plan to support it. Since Harbor leverages the beego, distribution & gocraft to manage cache for UI session, blob & manifest infor and jobs infor, it has to make these components to support redis cluster, than we can do it.

**Reply by w0aw** [reply]
@wy65701436 any update on this!

**Reply by SanduDS** [reply]
any update please

**Comment by ajaythecoder** [comment]
any update here, can we deploy aws elasticache with clustermode disabled? that works?

**Reply by ajaythecoder** [reply]
@wy65701436
